Source/Core/DolphinQt/Debugger/CodeWidget.cpp

```diff
@@ -243,15 +267,64 @@ void CodeWidget::OnPPCSymbolsChanged()
   }
 }
 
+void CodeWidget::OnSaveAddress()
+{
+  if (Core::GetState(m_system) != Core::State::Paused)
+    return;
+
+  u32 addr = m_code_view->GetAddress();
+  const QString address = QString::number(addr, 16);
+
+  Core::CPUThreadGuard guard(m_system);
+  std::string instruction = m_system.GetPowerPC().GetDebugInterface().Disassemble(&guard, addr);
+
+  if (instruction == "(No RAM here)" || instruction == "<unknown>")
```

Comment:
If you're going to match on exact text like this, please create a constant for the text and make both the caller and callee use the constant. That way we avoid things breaking if anyone wants to change the wording of the text in the future, or if anyone wants to make it translatable.

Reply:
I noticed one scenario was already handled by the If paused check, so I just did a check for if it is valid ram to cover the No Ram string.  So I don't need to check those strings after all.